src/modules/noit_lua_noit_binding.c

```diff
@@ -303,9 +225,16 @@ noit_message_index_func(lua_State *L) {
         break;
       }
       return 1;
+    case 'w':
+      if(!strcmp(k, "whence")) {
+        lua_pushinteger(L, msg->value.whence_ms/1000);
```

Comment:
is there a reason we hand this back as integer seconds?  That seem incongruent with the way reconnoiter looks at times (which are either timevals or milliseconds since epoch).  I think this should come back as either milliseconds or as two return values: seconds and microseconds (like timevals)